src/hotspot/share/runtime/globals.hpp

```diff
@@ -513,14 +513,14 @@ const size_t minimumSymbolTableSize = 1024;
           "Print additional debugging information from other modes")        \
                                                                             \
   /* SapMachine 2019-02-20 : stathist */                                    \
-  product(bool, EnableStatHist, true,                                       \
-          "Enable Statistics history")                                      \
+  product(bool, EnableVitals, true,                                         \
+          "Enable recording of vitals")                                     \
```

Comment:
* I would use the same word as below: "Enable __sampling__ of vitals"
  (or is the sampling on even if this flag is off?)
* I would give a short explanation of what vitals are:
"Enable sampling of vitals: memory, heap and cpu utilization" (Please use your own text ... this is only intended as example).

Reply:
Thanks Goetz. All good proposals. Will post update.